src/components/ExpensiTextInput/BaseExpensiTextInput.js

```diff
@@ -187,24 +200,41 @@ class BaseExpensiTextInput extends Component {
                                     />
                                 </>
                             ) : null}
-                            <TextInput
-                                ref={(ref) => {
-                                    if (typeof innerRef === 'function') { innerRef(ref); }
-                                    this.input = ref;
-                                }}
-                                // eslint-disable-next-line
-                                {...inputProps}
-                                value={value}
-                                placeholder={(this.state.isFocused || !label) ? placeholder : null}
-                                placeholderTextColor={themeColors.placeholderText}
-                                underlineColorAndroid="transparent"
-                                style={[inputStyle, !hasLabel && styles.pv0]}
-                                multiline={multiline}
-                                onFocus={this.onFocus}
-                                onBlur={this.onBlur}
-                                onChangeText={this.setValue}
-                                onPressOut={this.props.onPress}
-                            />
+                            <View style={[styles.flexRow, styles.expensiTextInputIconWrapper]}>
```

Comment:
This PR will bring back the issue that we recently fixed https://github.com/Expensify/App/issues/4515.
When you click on the highlighted area
![issue](https://user-images.githubusercontent.com/24370807/139657639-bed54d71-f269-4cb0-87a8-7181ee0600da.png)